tests/TestHelpers/GraphHelper.php

```diff
@@ -67,6 +67,13 @@ public static function getShareIdRegex(): string {
 		return self::getUUIDv4Regex() . ':' . self::getUUIDv4Regex() . ':' . self::getUUIDv4Regex();
 	}
 
+	/**
+	 * @return string
+	 */
+	public static function getFileIdRegex(): string {
+		return self::getUUIDv4Regex() . '\\\$' . self::getUUIDv4Regex() . '!' . self::getUUIDv4Regex();
```

Comment:
```suggestion
		return self::getSpaceIdRegex() . '!' . self::getUUIDv4Regex();
```
IMO, it will be clearer if the File ID combines space ID and unique uuid. 
I might be wrong.